src/Db/Dump/PKSimDB_dump.txt

```diff
@@ -80950,7 +80950,7 @@ INSERT INTO tab_processes VALUES('ActiveBiDirectionalSpecificInterstitialIntrace
 INSERT INTO tab_processes VALUES('ActiveBiDirectionalSpecificBloodCellsPlasma','PROCESS',1,'INDIVIDUAL_ACTIVE_PROCESS','Undefined','BiDirectional','TRANSPORT',0);
 INSERT INTO tab_processes VALUES('ActiveBiDirectionalSpecificBloodCellsPlasma_MM','PROCESS',1,'SIMULATION_ACTIVE_PROCESS','MM','BiDirectional','TRANSPORT',0);
 INSERT INTO tab_processes VALUES('ActiveBiDirectionalSpecificLumenMucosaIntracellular','PROCESS',1,'INDIVIDUAL_ACTIVE_PROCESS','Undefined','BiDirectional','TRANSPORT',0);
-INSERT INTO tab_processes VALUES('ActiveBiDirectionalSpecificLumenMucosaIntracellular_MM','PROCESS',1,'SIMULATION_ACTIVE_PROCESS','MM','BiDirectional','TRANSPORT',0);
+INSERT INTO tab_processes VALUES('ActiveBiDirectionalSpecificLumenMucosaIntracellular_MM','PROCESS',1,'SIMULATION_ACTIVE_PROCESS','MM','BiDirectional','TRANSPORT',1);

```

Comment:
Also set the parameter to 1... Why is this required?

Reply:
Because we need ProcessRate parameters of all transports (passive and active) between lumen and mucosa to calculate the fraction absorbed.
(dMassAbsorbed/dt is calculated as sum of all process rates)